src/test/mir-opt/inline/inline-any-operand.rs

```diff
@@ -1,4 +1,4 @@
-// compile-flags: -Z span_free_formats
+// compile-flags: -Z span_free_formats -Zenable-experimental-unsound-mir-opts
```

Comment:
why is this needed? From the test description it doesn't look like this needs copy prop or simplify_try